### PR TITLE
Restoring OSGi compatibility after migration to a fully Maven-based b…

### PR DESCRIPTION
**Pull request by cfiehe**
…uild process. This commit also fixes issue #162.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <packaging>jar</packaging>
 
   <name>RabbitMQ Java Client</name>
-  <description>RabbitMQ Java client</description>
+  <description>The RabbitMQ Java client library allows Java code to interface to AMQP servers.</description>
```
Comment (michaelklishin) [inline pom.xml]: Can we please back this change out?

File: pom.xml [hunk 2]
```diff
@@ -43,6 +43,11 @@
     <url>https://github.com/rabbitmq/rabbitmq-java-client.git</url>
     <connection>scm:git:https://github.com/rabbitmq/rabbitmq-java-client.git</connection>
   </scm>
+  
+  <organization>
+    <name>Rabbit Technologies Ltd.</name>
```
Comment (michaelklishin) [inline pom.xml]: Rabbit Technologies has been acquired by SpringSource/VMware in 2010. Please use `Pivotal Software, Inc`.

File: pom.xml (hunk 1 above)
Comment (michaelklishin) [inline pom.xml]: Or at least replace "AMQP servers" with "RabbitMQ" because "AMQP" means more than one protocol these days.

**Comment by michaelklishin** [comment]
Thank you!

File: pom.xml (hunk 1 above)
Comment (cfiehe) [inline pom.xml]: I have just copied the description from here: https://www.rabbitmq.com/java-client.html. But no problem, I will replace "AMQP servers" with "RabbitMQ". The description in the pom gets used as "Bundle-Description" in the OSGi manifest file and things look better with a proper outline.

File: pom.xml (hunk 2 above)
Comment (cfiehe) [inline pom.xml]: I was not aware of that fact. It will be changed immediately.

**Comment by cfiehe** [comment]
The pull request has been updated.

**Comment by michaelklishin** [comment]
Thank you once again. We will be backporting all the Maven bits to `stable` and this client releases will be decoupled from RabbitMQ releases fairly soon. No promises on whether this will make it into `3.6.6` but chances are it will.

**Comment by cfiehe** [comment]
That was pretty fast ;-) Thank you very much for the merge.

No problem.., From my side, I can stay on version 3.6.3 until the fix gets it into the release cycle.

**Pull request merged**